staker/types.go

```diff
@@ -464,6 +462,19 @@ func parseWatchStakingRequest(
 	currentParams *cl.StakingParams,
 	network *chaincfg.Params,
 ) (*stakingRequestedEvent, error) {
+	// TODO: This check re-implements whole babylon validation logic. We should
```

Comment:
Maybe add "Todo(konrad)", I feel like only putting todo gets lost quickly and setting your name/username next it adds move ownership that this will eventually get done